hbase-rest/src/main/java/org/apache/hadoop/hbase/rest/model/CellModel.java

```diff
@@ -58,10 +61,11 @@
  * </pre>
  */
 @XmlRootElement(name = "Cell")
-@XmlAccessorType(XmlAccessType.FIELD)
+@XmlAccessorType(XmlAccessType.NONE)
```

Comment:
Why we need this change?

Reply:
XmlAccessType.FIELD implicitly adds every field of the method to the XML.

So if we kept XmlAccessType.FIELD, we'd have to annotate the transient fields.
Since we are are already annotating every field that we wanto to marshall, it's simpler to just change the class-level setting.